include/envoy/runtime/runtime.h

```diff
@@ -266,7 +266,7 @@ class Loader {
 public:
   virtual ~Loader() = default;
 
-  using ReadyCallback = std::function<void()>;
+  using ReadyCallback = std::function<void(const std::string)>;
```

Comment:
Passing `const std::string` is very unusual c++; it makes a copy that can't be modified in the function. Prefer passing `absl::string_view`, and don't use `const` for value params. This goes for all instances below.